accord-core/src/test/java/accord/utils/Property.java

```diff
@@ -445,6 +446,7 @@ public <State, SystemUnderTest> void check(Commands<State, SystemUnderTest> comm
             {
                 State state = null;
                 List<String> history = new ArrayList<>(steps);
+                LongArrayList historyTiming = stepTimeout == null ? null : new LongArrayList();
```

Comment:
this is 100% not required, but it was helpful to know how long things were taking... are we relying on retries?  this will help make that clear